### ISSUE TITLE
security scan failing

### ISSUE THREAD

**Issue by ghafran** [dudleycarr/nsqjs#115]
After scanning nsqjs module.
```
npm install nsqjs --save
npm install nsp --global
nsp check
```
Getting these issues

![screen shot 2017-01-02 at 3 06 44 pm](https://cloud.githubusercontent.com/assets/769045/21595549/18b37c24-d0fd-11e6-8c05-1dc688c95297.png)

Looks like its using outdated modules....

**Comment by dudleycarr** [comment]
Fixed in #116

**Comment by ghafran** [comment]
Passes with flying colors!
Thanks.
